tools/hangar/__snapshots__/test_corpus/valid/print.test.w_test_sim.md

```diff
@@ -8,6 +8,8 @@ pass ┌ print.test.wsim » root/env0/test:log1
      │ inflight log 1.1
      └ inflight log 1.2
 pass ┌ print.test.wsim » root/env1/test:log2
+     │ inflight log 1.1
+     │ inflight log 1.2
```

Comment:
Looks like the logs are getting messed up now that the tests are running in parallel.

Maybe keep them running serially for this PR and work on parallelizing on a different one.